### PR TITLE
WIP - docs

### PR DESCRIPTION
**Pull request by ghoneycutt**
need to add bind::forwarders

### CODE REVIEW

**Comment by ghoneycutt** [comment]
@Phil-Friderici realized I've written a bunch already, but wasn't quite done. Could you pick it up from here.

File: README.md [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
 
-Manage bind9. This module's approach is to manage the configuration of bind9
-while not managing the data in the actual zones, which is up to you. Recommend
-keeping zone data in another repo and having a process sync that data to your
-bind servers.
-
-===
-
-### Table of Contents
-1. [Compatibility](#compatibility)
```
Comment (ghoneycutt) [inline README.md]: This should probably be back and have entries for each class/define, since the README is going to be huge.

Comment (Phil-Friderici) [inline README.md]: TOC added in https://github.com/ghoneycutt/puppet-module-bind/pull/21

**Comment by ghoneycutt** [comment]
@Phil-Friderici please close if all of these params are in PR #21

**Pull request merged**